sdks/python/apache_beam/io/filebasedsource.py

```diff
@@ -144,22 +145,29 @@ def open_file(self, file_name):
         compression_type=self._compression_type)
 
   @staticmethod
-  def _estimate_sizes_in_parallel(file_names):
+  def _estimate_sizes_in_parallel(file_names, pattern=None):
```

Comment:
Rename to '_estimate_sizes_of_files' since 'in_parallel' part does not apply if a pattern is specified.